src/morph/src/Request.re

```diff
@@ -1,22 +1,15 @@
-type t = {
+type t('body) = {
   target: string,
   meth: Method.t,
   headers: list((string, string)),
-  read_body: unit => Lwt.t(string),
+  body: 'body,
```

Comment:
I'm not sure about this change, it was done when I was trying to remove Lwt

Reply:
We could revert this and add a optional labeled argument to get more body types. This way we can have streams that would be nice for things like file upload

Reply:
To sum up our offline conversation - I think this change is correct. It allows the server implementation to define the possible body types giving the users full freedom as to how the data representation should look like. One could imagine cases where the server implementation parses from the blob it receives to a more concrete, application specific type, for instance `Yojson.t`. It's a low level primitive and it provides a lot of freedom as it should.

Reply:
Yes, this is basically good to go